static/src/javascripts/projects/admin/bootstraps/commercial-browser-performance.js

```diff
@@ -0,0 +1,104 @@
+// @flow
+/* global $, Epoch*/
+import config from 'lib/config';
+import fetchJson from 'lib/fetch-json';
+import template from 'lodash/utilities/template';
+import countBy from 'lodash/collections/countBy';
+import find from 'lodash/collections/find';
```

Comment:
Any chance to replace the lodash method with [Array.find()](https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Global_Objects/Array/find?v=control)?